src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/GuideToPGENExtract.md

```diff
@@ -0,0 +1,85 @@
+# Extracting PGEN from GVS
+
+## The PGEN format
+PGEN is a format written for and used by version 2 of [PLINK](https://www.cog-genomics.org/plink/2.0/).  ***IT IS VERY IMPORTANT TO NOTE THAT VERSION 2 OF PLINK IS STILL IN ALPHA AND THE PGEN FORMAT IS STILL SUBJECT TO CHANGE.***  The format comprises 3 file types (or actually sometimes 4):
+1. A `.pgen` file.  This is a binary file that stores a mapping of samples and sites to genotypes in a very cleverly compressed way that I can't explain super well because it's complicated.
+2. A `.pvar` file. This is essentially a sites-only VCF.  It has information for each site referenced in the `.pgen` file.  PLINK also provides an option to produce/use a zstd compressed version of the file (`.pvar.zst`), and we have opted to write that for performance purposes.
+3. A `.psam` file.  This is a plaintext file that contains a list of samples referenced in the `.pgen` file.  It also optionally includes some phenotype data.
+4. Optionally, a `.pgi` file.  Typically, a `.pgen` file has an index at the top.  Optionally, PLINK supports using a `.pgen` file with an index in a separate `.pgi` file.
+
+The PGEN format does not store all of the information that a VCF has.  It leaves out a lot of the fields and annotations you can store in a VCF.  As a result of this and the clever compression in the `.pgen` file, these files are typically much smaller than equivalent VCFs.
+
+For more information on the PGEN file format, see the official spec [here](https://github.com/chrchang/plink-ng/blob/master/pgen_spec/pgen_spec.pdf).
+
+## The code
+The code for the PGEN extract can be divided into 3 parts:
+1. The PGEN-JNI, a C++/JNI library that handles writing HTSJDK VariantContext objects to PGEN files,
+2. ExtractCohortToPgen, a GATK tool based on ExtractCohortToVcf that processes VariantContexts and passes them to PGEN-JNI for writing, and
+3. GvsExtractCallsetPgenMerged, a WDL workflow based on GvsExtractCallset that uses ExtractCohortToPgen to write a series of PGEN files and then merges them by chromosome.
+
+### Part 1: PGEN-JNI
+The PGEN-JNI library was written by Chris Norman of the GATK Engine Team and lives [here](https://github.com/broadinstitute/pgen-jni).  It is written primarily in C++ for performance purposes and also compatibility with the pgenlib library (part of the [plink repo](https://github.com/chrchang/plink-ng/tree/master)).  It builds on top of pgenlib to provide a writer for creating PGEN files and writing to them from HTSJDK VariantContext objects.
+
+PGEN-JNI is compatible with Linux and macOS.
```

Comment:
Though ["MacOS aarm64 (sic) is not yet supported"](https://github.com/broadinstitute/pgen-jni?tab=readme-ov-file#supported-runtime-platforms), which I believe is what most of us are using.

Reply:
Ooo, good point.  I'll update that